tutorials/quickstart/Solution/Billing/Program.cs

```diff
@@ -1,34 +1,37 @@
 ﻿using System;
 using System.Threading.Tasks;
+using Microsoft.Extensions.Hosting;
 using NServiceBus;
 
 namespace Billing
 {
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("Style", "IDE0008:Use explicit type")]
```

Comment:
Why did these get added? Do you have a local editorconfig trying to force explicit variable types?

Reply:
I was getting messages to use the explicit type.  I can remove the suppression if was just me.  They were Info messages anyway.